src/pages/_app.js

```diff
@@ -27,7 +27,6 @@ try {
 
 const MyApp = (props) => {
   const { Component, pageProps, err } = props
-
```

Comment:
Leave blank lines before comments for readability. (We can add [this ESLint rule](https://eslint.org/docs/rules/lines-around-comment) at some point.)